utils/styleguide/styleguide.base.config.js

```diff
@@ -44,6 +44,10 @@ const defaultStyleguideConfig = {
       sidebarBackground: zdColorGrey100,
       name: zdColorKale400,
       type: zdColorRed600
+    },
+    fontSize: {
+      base: 14,
+      text: 14
```

Comment:
Does it work to use `zdFontSizeMd` here?

Reply:
Good call. I've also changed the styles to use `jsx-global` style inclusion as defined in their styleguidist cookbook.

Should be a little more clear.